src/awkward/_v2/contents/indexedarray.py

```diff
@@ -1261,8 +1270,8 @@ def _to_nplike(self, nplike):
         return IndexedArray(
             index,
             content,
-            identifier=self.identifier,
-            parameters=self.parameters,
+            identifier=self._identifier,
+            parameters=self._parameters,
```

Comment:
Good catch: we should prefer private variables within the class (just to have a rule to follow). In particular, `self.parameters` creates a dict when one is not needed.